config/routes.rb

```diff
@@ -57,7 +57,12 @@
       resources :courses, only: %i[index edit update]
       resources :locations
     end
-    resources :users, only: %i[index show new create destroy]
+
+    resources :users do
+      scope module: :users do
+        get :providers, on: :member, to: "providers#show"
```

Comment:
```suggestion
resources :providers, only: :show
```

then you don't have to map?

Reply:
Good spot, commited 👍